src/core/utils.js

```diff
@@ -138,6 +140,11 @@ function fromTimestamp(timestamp) {
   return Math.round(timestamp_ / 1000) - 0x386D4380;
 }
 
+function sha256(bytes) {
```

Comment:
Could just use hash.js I guess, or ripple-address-codec, which is x-address-codec bound setup with hash.js and ripple as the default alphabet for exported encode/decode. Leads the question:  Do we want to move away from sjcl in a big bang, or over a few pull requests?